src/components/Form/UploadDocumentForm.jsx

```diff
@@ -32,58 +29,71 @@ import FormSection from './FormSection';
  */
 
 const UploadDocumentForm = () => {
-  const { session } = useSession();
   const { state, dispatch } = useStatusNotification();
   const [expireDate, setExpireDate] = useState(null);
   const [docDescription, setDocDescription] = useState('');
   const [docType, setDocType] = useState('');
+  const [verifyFile, setVerifyFile] = useState(false);
+  const [file, setFile] = useState(null);
+  const [inputKey, setInputKey] = useState(false);
+  const { addDocument, replaceDocument } = useContext(DocumentListContext);
 
   const handleDocType = (event) => {
     setDocType(event.target.value);
   };
 
-  // Initialized state for file upload
-  const handleFileChange = (event) => {
-    if (event.target.files.length === 1) {
-      dispatch({ type: 'SET_FILE', payload: event.target.files[0] });
-    } else {
-      dispatch({ type: 'CLEAR_FILE' });
-    }
-  };
+  // Custom useField hook for handling form inputs
 
   const clearInputFields = () => {
-    dispatch({ type: 'CLEAR_FILE' });
-    dispatch({ type: 'CLEAR_VERIFY_FILE' });
-    dispatch({ type: 'CLEAR_PROCESSING' });
+    setVerifyFile(false);
+    setDocType('');
+    setFile('');
+    setInputKey(!inputKey); // clears file by forcing re-render
     setDocDescription('');
     setDocType('');
-    setExpireDate(null);
+    setExpireDate('');
```

Comment:
The Dayjs value used in the date picker is a Dayjs object. Think it'll be more appropriate to use a null.

Reply:
fixed